app/single-instance.js

```diff
@@ -43,9 +50,14 @@ export default function () {
       getMainWindow = require('./app').getMainWindow
       net
         .createServer(connection => {
-          connection.on('data', () => {
+          connection.on('data', data => {
             const mainWindow = getMainWindow()
             if (mainWindow) {
+              const file = data.toString()
+              if (file.endsWith('.rep')) {
+                mainWindow.webContents.send('openReplay', file)
```

Comment:
This, too, demonstrates the risk of doing this in a bunch of different places (here you aren't using the type-safe IPC stuff, but in the other place you are), definitely should be shared code